lib/perf_hooks.js

```diff
@@ -448,6 +453,26 @@ class Performance {
     return ret;
   }
 
+  eventLoopUtilization(util) {
+    const ls = nodeTiming.loopStart;
+
+    if (ls <= 0) {
+      return { idle: 0, active: 0, utilization: 0 };
+    }
+
+    if (!util) {
+      const idle = nodeTiming.idleTime;
+      const active = performance.now() - ls - idle;
+      return { idle, active, utilization: active / (idle + active) };
+    }
+
+    const idle = nodeTiming.idleTime;
+    const active = performance.now() - ls - idle;
```

Comment:
`idle` and `active` are defined identically in the if block above. Why not just define them _before_ the block and use the same values in both places? 🤔